go/main/main.go

```diff
@@ -68,19 +68,24 @@ func main() {
 
 	// initialize the ubirch protocol sim interface
 	err = sim.Init(PIN)
+	// alternatively you can skip bootstrapping and authorize with PIN from config file with the following line
+	//err = sim.Init(conf.Sim.Pin)
```

Comment:
Does this fail if there is no PIN entry in the config? (it should)

Reply:
pin is acquired via bootstrapping. took this bit out where the pin is retrieved from config